app.js

```diff
@@ -24,12 +25,18 @@ app.use(bodyParser.urlencoded({ extended: false }));
 app.use(cookieParser());
 app.use(express.static(path.join(__dirname, 'public')));
 app.use(session({ secret: 'keyboard cat', cookie: { maxAge: 60000 }}))
-app.use(skipRoutes(['/users/auth', '/users/login', '/lti'], redirectAnonymous));
+app.use(skipRoutes(['/', '/users/auth', '/users/login', '/lti'], redirectAnonymous));
+
+app.use((req, res, next) => {
```

Comment:
Since res parameter is not used, it should be replaced with _ to follow convention.